pkg/services/cosmosdb/catalog.go

```diff
@@ -70,11 +64,11 @@ func (m *module) GetCatalog() (service.Catalog, error) {
 						"MongoDB",
 					},
 				},
-				m.serviceManager,
```

Comment:
Github's not letting me comment on lines that weren't changed, but consistent with [this comment](https://github.com/Azure/open-service-broker-azure/pull/332/files#r174914255), about a dozen lines up from here, could we consider a name change to "azure-cosmosdb-mongo-account"?